docs/source/examples/fastsiam.rst

```diff
@@ -0,0 +1,47 @@
+.. _fastsiam:
+
+FastSiam
+========
+
+Example implementation of the FastSiam architecture.
```

Comment:
We now usually add a short description of the model. See for example: https://docs.lightly.ai/self-supervised-learning/examples/mae.html. Feel free to make a follow-up issue to add it later.